backend/src/main/java/com/bakdata/conquery/sql/execution/DefaultResultSetProcessor.java

```diff
@@ -48,11 +50,12 @@ public Boolean getBoolean(ResultSet resultSet, int columnIndex) throws SQLExcept
 
 	@Override
 	public Number getDate(ResultSet resultSet, int columnIndex) throws SQLException {
-		Date date = resultSet.getDate(columnIndex);
-		if (date == null) {
+		String dateString = resultSet.getString(columnIndex);
+		if (dateString == null) {
 			return null;
 		}
-		return date.toLocalDate().toEpochDay();
+		DateReader dateReader = config.getLocale().getDateReader();
```

Comment:
Wir müssen noch schauen, wie wir die locale auch in der Session konfigurieren oder? Das muss schon zusammenpassen, sonst reden SQL und Conquery aneinander vorbei :')

Reply:
Auf jeden Fall, ich denke dass wir das in diesem Zuge eh lösen müssen #3492.